src/components/Avatar/index.js

```diff
@@ -62,13 +62,16 @@ const Avatar = props => {
 
   return (
     <div className={`${styles.avatar} ${className}`}>
-      <Circle
-        border={border}
-        size={size}
-        color="white"
-        input={img ? img : hash}
-        type={img ? 'img' : 'blocky'}
-      />
+      {/* empty div used to avoid stretching inside flexbox */}
+      <div>
+        <Circle
+          border={border}
+          size={size}
+          color="white"
+          input={img ? img : hash}
+          type={img ? 'img' : 'blocky'}
+        />
+      </div>
```

Comment:
Is there a way to achieve this without adding an extra div that serves no other purpose? Perhaps by altering the CSS of the avatar component instead of the actual markup, since it's a styling issue and not a markup issue?